### PR TITLE
Prototype of a model for the register of Ukrainian corrupters

### PR DESCRIPTION
**Pull request by EmmaSamoilova**
NOT FOR MERGE
Prototype of a model for the register of Ukrainian corrupters
https://corruptinfo.nazk.gov.ua/ep/swagger/ui/index#!/Corrupt/Corrupt_Get

### CODE REVIEW

File: corrupter_register/models.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    #     choices=PUNISHMENT_TYPES,
+    #     max_length=2,
+    #     help_text='Punishment type. Can be \'court decision\' or \'disciplinary action\'.'
+    # )
+    # entity_type = models.CharField(
+    #     _('entity type'),
+    #     choices=ENTITY_TYPES,
+    #     max_length=2,
+    #     help_text='Entity type. Can be \'Individual entrepreneur\' or \'Legal entity\'.'
+    # )
```
Comment (EmmaSamoilova) [inline corrupter_register/models.py:68]: I don't know how better to make these two fields. As "choices" or as a separate model as here https://corruptinfo.nazk.gov.ua/ep/swagger/ui/index#!/Corrupt/Corrupt_Get

File: corrupt_person_register/models.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+        help_text='The address of registration of the legal entity at the time of the offense: the name of '
+                  'the region/city of national importance'
+    )
+    addr_str = models.CharField(
+        _('full address'),
+        max_length=200,
+        null=True,
+        blank=True,
+        help_text='The address of registration of the legal entity at the time of the offense: district, town, '
+                  'street, house, premises in the form of a line.'
```
Comment (IegorSoboliev) [inline corrupt_person_register/models.py:171]: Maybe, you will look at our DRV models (DrvRegion, DrvStreet, etc)?

File: corrupt_person_register/models.py [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    )
+    short_name = models.CharField(
+        _('Short name of the legal entity'),
+        max_length=20,
+        db_index=True,
+        null=True,
+        blank=True,
+        help_text='Abbreviated name of the legal entity at the time of the offense.'
+    )
+    full_name = models.CharField(
```
Comment (IegorSoboliev) [inline corrupt_person_register/models.py:181]: full name is https://en.wiktionary.org/wiki/full_name#:~:text=full%20name%20(plural%20full%20names,%3A%20see%20full%2C%E2%80%8E%20name.

File: corrupt_person_register/models.py [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    codex_article_nazk_id = models.PositiveIntegerField(
+        _('codex_article_nazk_id'),
+        unique=True,
+        help_text='Identifier of the article according to which the person was prosecuted.'
+    )
+    codex_article_name = models.TextField(
+        _('codex article name'),
+        help_text='The name of the article under which the person was prosecuted.'
+    )
+
```
Comment (IegorSoboliev) [inline corrupt_person_register/models.py:18]: maybe we need also verbose names)

File: corrupt_person_register/models.py [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+        help_text='Identifier of the sphere of activity of an individual at the time of the offense'
+    )
+    activity_sphere_name = models.CharField(
+        _('name of activity sphere'),
+        max_length=200,
+        help_text='Name of the sphere of activity of an  individual at the time of the offense.'
+    )
+
+
+class CorruptPerson(DataOceanModel):
```
Comment (IegorSoboliev) [inline corrupt_person_register/models.py:71]: Corrupt?

File: corrupt_person_register/models.py (hunk 3 above)
Comment (EmmaSamoilova) [reply corrupt_person_register/models.py:181]: I named it like here https://corruptinfo.nazk.gov.ua/ep/swagger/ui/index#!/Corrupt/Corrupt_Get but a bad name, I agree

File: corrupt_person_register/models.py (hunk 5 above)
Comment (EmmaSamoilova) [reply corrupt_person_register/models.py:71]: So called on techtalk.

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**